src/app/home/avatar-generator/attribute-selector/attribute-selector.component.html

```diff
@@ -0,0 +1,35 @@
+<!-- Rounded dialog -->
+<div class="card nes-container with-title" (click)="decoratorModal.open()">
+  <p class="title">{{ decorator?.name }}</p>
+  <div [innerHTML]="getIcon() | safeHtml"></div>
+</div>
+
+<div #decoratorModal=modalDirective wgjModal class="modal nes-dialog is-rounded">
+  <ng-container *ngIf="decorator">
+    <button type="button" class="nes-btn is-error close-btn pixel-text white-text" (click)="decoratorModal.close()">
```

Comment:
```suggestion
    <button type="button" class="nes-btn is-error close-btn white-text" (click)="decoratorModal.close()">
```
We could safely remove the `pixel-text` class from this element, and from the label element on line 19.